.github/workflows/ci.yml

```diff
@@ -40,14 +40,14 @@ jobs:
         run: |
           conan create --version 0.11.0.3 zenohc-tmp/prebuilt
           conan create --version 0.11.0 zenohc-tmp/prebuilt
-          conan create --version 1.0.0-rc4 zenohc-tmp/prebuilt
```

Comment:
Keep this line since it is still in the conandata file. We should build all published releases in the CI